aten/src/ATen/native/native_functions.yaml

```diff
@@ -1180,6 +1180,7 @@
     CPU: logspace_cpu_out
     CUDA: logspace_cuda_out
 
+# log_softmax doesn't follow kwargs-only convention because BC-breaking when loading jit models that use this operator.
```

Comment:
I don't know what "kwargs-only convention" is, but maybe that's just me

Reply:
re-worded, let me know if that's clearer.

Reply:
Ah yes. Much clearer now, thanks